pallets/grants/src/tests.rs

```diff
@@ -285,6 +286,282 @@ fn cancel_auto_claim_recipient_funds_and_wire_the_rest() {
 	});
 }
 
```

Comment:
I might be pushing it, but it might be a good thing to get tests for the `ensure_root` statements as to ensure they do not get removed by mistake. Meaning two tests: `remove_bridge_ensures_root` and `set_bridge_ensures_root` (or whichever name is best).

Reply:
Good suggestion. I will be adding them.

Reply:
Done